lib/msgr/client.rb

```diff
@@ -59,8 +59,6 @@ def start
 
         log(:debug) { "Start on #{uri}..." }
 
-        @routes << config[:routing_file] if config[:routing_file].present?
-        @routes.reload
```

Comment:
I had to inject the routes and move their initialization to the railtie, so that I can access them without starting to consume messages.

Reply:
Oh wait, that has changed now that `drain` is encapsulated within `Client`. Hmm. :thinking: